funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_manager.py

```diff
@@ -543,6 +552,8 @@ def cli_run():
     parser.add_argument("--worker_mode", default="singularity_reuse",
                         help=("Choose the mode of operation from "
                               "(no_container, singularity_reuse, singularity_single_use"))
+    parser.add_argument("--container_cmd_options", default="",
```

Comment:
If the user gives a `container_cmd_options='--key=foo --key2=foo2'` would this still work? Have we tested the escape logic to make sure that strings with space will still work?

Reply:
I just looked back. I already tested this and ensured the space is escaped by putting a single quote around the `funcx-manager` command: `"--container_cmd_options='{container_cmd_options}' "`. I will comment on this line below.